src/core.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Map, containing all registered modules of the bot.
+ *
+ * @type {Map.<Object>}
+ * @constant
+ */
+const MODULES = new Map();
```

Comment:
I think this should be lowercase, since this is mutable variable anyway. What do you think?

Reply:
It depends on a codestyle we're using. I find an information about the mutable constants situation only in a Google Guileline, but there are several options:
* [idiomatic.js|https://github.com/rwaldron/idiomatic.js/tree/master/translations/ru_RU]
Symbolic constants always UPPER_CASE;
* [dojo|https://dojotoolkit.org/reference-guide/1.10/developer/styleguide.html]
Constants either CamelCase (like ClassName) or UPPER_CASE;
* [jQuery|http://contribute.jquery.org/style-guide/js/#naming-conventions]
No info about `const`, always mixedCamelCase;
* [Google|https://google.github.io/styleguide/javascriptguide.xml]
Use UPPER_CASE for constant _values_, don't use `const` keyword because of `IE`, use `@const` in JSDoc instead;
* [Airbnb|https://github.com/airbnb/javascript]
Always use mixedCamelCase, even with `const` vars.

Personaly, I didn't want to make `MODULES` uppercase, so I can suggest to use Google's styleguide. If we approve it (or some other styleguide, or even develop smth different), I will make it lowercase and use "mutable const is lowercase" as a rule of thumb.

Reply:
Decided to use Airbnb codestyle.